### DISCUSSION TITLE
Rename priorities

### DISCUSSION THREAD

**Discussion by citiz3n404** [ransome1/sleek#468] [Ideas]
I would be great if we could rename the priorities or enable a directory base sorting like "Waiting" "Current" "Planned" "Today"

**Comment by mshuflin** [comment]
That does not match the todo.txt spec https://github.com/todotxt/todo.txt why don't you use projects or contexts for that?

**Comment by ransome1** [comment]
@citiz3n404 , @mshuflin is right, be cannot change how the priorities work.

But in the near future sleek will provide "tags", so you can add something like "#Waiting" to your todos and use these tags for grouping and sorting.
